test/templates/server.spec.ts

```diff
@@ -71,7 +81,7 @@ beforeAll(() => {
 })
 
 describe('the netlify next server', () => {
-  it('does not revalidate a request without an `x-prerender-revalidate` header', async () => {
+  it.skip('does not revalidate a request without an `x-prerender-revalidate` header', async () => {
```

Comment:
this is all part of the on-demand revalidation that needs to be completed reworked 😬
i've left the tests there in case we need to reference them in future but it'll need to be revisited when we look at on-demand revalidation next